test/integration/job/job_test.go

```diff
@@ -133,8 +133,8 @@ func TestJobPodFailurePolicyWithFailedPodDeletedDuringControllerRestart(t *testi
 					},
 				},
 			},
-			Parallelism: ptr.To(int32(count)),
-			Completions: ptr.To(int32(count)),
+			Parallelism: ptr.To[int32](count),
+			Completions: ptr.To[int32](count),
```

Comment:
we do not need the casting here now, right?

Reply:
Yes, that's right.
That is this PR's motivation.

Reply:
That is not runnable since the ptr doesn't have knowledge of types.
But, I guess that the below code is more better. So, let me select the below here.

```suggestion
			Parallelism: &count,
			Completions: &count,
```

Reply:
@mimowo @atiratree I selected the approach, "&count".
If you have another better approach, please let me know. Thanks.

Reply:
lgtm

FIY, there is a type information from the variable so empty `ptr.To(count)` should work. Nevertheless `&count` is even better

Reply:
Oh, you're right. I was missing anything.
@mimowo suggestion is executable since we already passed the typed knowledges in the `count := int32(3)`.